content/en/docs/tasks/administer-cluster/dns-custom-nameservers.md

```diff
@@ -51,9 +51,9 @@ lookups (PTR records).
 
 ## Inheriting DNS from the node
 
-When running a Pod, kubelet prepends the cluster DNS server and searches
-paths to the node's DNS settings.  If the node is able to resolve DNS names
-specific to the larger environment, Pods should also be able to resolve.
+If a pod's `dnsPolicy` is set to "`default`", it inherits the name resolution
+configuration from the node that the pods run on. If the node is able to resolve
```

Comment:
you dropped the part about "prepends cluster DNS server"

Reply:
Yes, for Pods with DNSPolicy=ClusterFirst, cluster DNS server is the only NS, for Pods with DNSPolicy=default, it inherits the name resolution configuration from the node. None of dns policies would prepend cluster DNS server and node's DNS server since kubernetes/kubernetes#18089.

Reply:
replace "If the node is able ...resolve." with "Pods DNS resolution should behave the same on the Node."

Reply:
sorry, I misread your previous sentence.

Reply:
Done. Thanks!